test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Projects/CpsPackageReferenceProjectProviderTests.cs

```diff
@@ -24,7 +22,7 @@ public CpsPackageReferenceProjectProviderTests(GlobalServiceProvider globalServi
 
         // As of October 2020, Service Fabric projects (sfproj) uses CPS, but does not support PackageReference. Make sure non-PR CPS projects do not use this project system.
         [Fact]
-        public async Task TryCreateNuGetProjectAsync_CpsProjectWithoutPackageReferencesCapability_ReturnsNull()
+        public async void TryCreateNuGetProject_CpsProjectWithoutPackageReferencesCapability_ReturnsNull()
```

Comment:
It looks like in https://github.com/xunit/xunit/commit/76d2f4e81ad1aafd137b52668e9bf33c24b698fb commit, xUnit removed support for async void tests. When we update the xUnit version to latest these tests may not work as expected.

Reply:
That was a mistake. Fixed now.